egs/wsj/s5/steps/nnet3/nnet3_train_lib.py

```diff
@@ -242,6 +319,53 @@ def GenerateEgs(data, alidir, egs_dir,
           egs_dir = egs_dir,
           egs_opts = egs_opts if egs_opts is not None else '' ))
 
+def GenerateEgsFromTargets(data, targets_scp, egs_dir,
```

Comment:
may be call it `GenerateEgsUsingTargets`. Also add a comment saying that this differs from the other method which uses alignments.